### PR TITLE
Register polo.is-a.dev

### PR DESCRIPTION
**Pull request by poloputoamo**
Register polo.is-a.dev with TXT record pointing to _value=vc-domain-verify=polo.is-a.dev,218d849c972de88cbdf4.

### CODE REVIEW

File: domains/polo.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "polo-1245-oficial",
+           "email": "81937184+polo-1245-oficial@users.noreply.github.com",
+           "discord": "725316907187568701"
+        },
+    
+        "record": {
+            "TXT": "_value=vc-domain-verify=polo.is-a.dev,218d849c972de88cbdf4"
```
Comment (21Z) [inline domains/polo.json:9]: TXT record alone does nothing... and we dont support vercel atm. (might soon tho)

**Review by 21Z — CHANGES_REQUESTED** [review]

**Comment by polo-1245-oficial** [comment]
can you check now? ;)

File: domains/polo.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "polo-1245-oficial",
+           "email": "81937184+polo-1245-oficial@users.noreply.github.com",
```
Comment (CuteDog5695) [inline domains/polo.json:4]: Please replace the noreply email with either your own email or leave it blank

File: domains/polo.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+           "discord": "725316907187568701"
+        },
+    
+        "record": {
+            "CNAME": "polo-1245-oficial.github.io"
```
Comment (CuteDog5695) [inline domains/polo.json:9]: Page seems to give a 404 error, if you have the site hosted then please send a preview of it

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/polo.json (hunk 3 above)
Comment (polo-1245-oficial) [reply domains/polo.json:8]: preview image

**Comment by polo-1245-oficial** [comment]
updated ;)

File: domains/polo.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+        "owner": {
+           "username": "polo-1245-oficial",
```
Comment (21Z) [inline domains/polo.json:3]:
```suggestion
           "username": "polo-1245-oficial",
           "email": "",
```

**Review by 21Z — CHANGES_REQUESTED** [review]

**Comment by polo-1245-oficial** [comment]
done

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**